a lot of providers documentation (#4287)
centreon/centreon-documentation: cloud/alerts-notifications/ticketing/providers/bmc-footprints-11.md

---
id: bmc-footprints-11
title: BMC Footprints 11
---

## Features information

| open ticket | close ticket (from Centreon to BMC Footprints) | Handle custom fields |
| -- | -- | -- |
| ✓ | ✘ | ✘ |

## Prerequisites

### Network Flow

| Source | Destination | Protocol/Port |
| -- | -- | -- |
| Centreon central server | BMC Footprints | TCP/443 (https) or TCP/80 (http) |

### Account

You need the following information

- Username
- Password

> The aformentioned account must be able to at least be able to open a ticket through the SOAP API action **MRWebServices**

Reviewer: ```suggestion
The aforementioned account must be able to at least be able to open a ticket through the SOAP API action **MRWebServices**
```

> Some tests commands that you can run from your Centreon central server are available in the [Test commands chapter](#test-commands)

Reviewer: ```suggestion
Some test commands that you can run from your Centreon central server are available in the [Test commands chapter](#test-commands)
```

## Sent data

This open ticket connector can send the following information when opening a ticket

- Project ID
- Priority
- Assignee
- Status

## Test commands

> The below Curl commands must be run from your central server. You need to replace everything between `<>` for example `<footprints_address>` may become **my_footprints.local**

Reviewer: ```suggestion
The Curl commands listed below must be run from your central server. You need to replace everything between `<>` for example `<footprints_address>` may be replaced with **my_footprints.local**
```

> In the --data part of the curl command, all `<UPPER CASE>` information must be replaced by their approriate values (therefore, `<TITLE>` must be replaced by "Test ticket" )

Reviewer: ```suggestion
In the --data part of the curl command, all `<UPPER CASE>` information must be replaced with their appropriate values (therefore, `<TITLE>` may be replaced with "Test ticket" )
```

### Open a ticket

> Keep in mind that the data in the below command is just an example and your footprints server may ask you to add mandatory data

Reviewer: ```suggestion
Keep in mind that the data in the command down below is just an example and your footprints server may ask you to add mandatory data
```
